index.src.html

```diff
@@ -1436,23 +1433,21 @@
 ## Defining extensions ## {#extension-specification}
 
 A definition of an extension must specify, at minimum, an extension identifier and an extension client argument sent via the
-{{getAssertion()}} or {{makeCredential()}} call (see below). Additionally, extensions may specify additional values in
-{{ClientData}}, `authenticatorData` (in the case of signature extensions), or both.
+{{getAssertion()}} or {{makeCredential()}} call. Additionally, extensions may specify additional values in {{ClientData}}, 
+`authenticatorData` (in the case of signature extensions), or both. Finally, if the extension requires any authenticator 
+processing, it must also specify an authenticator argument to be sent via the <a>authenticatorGetAssertion</a> or 
+<a>authenticatorMakeCredential</a> call.
 
-Note: An extension that does not define additions to {{ClientData}} nor `authenticatorData` is possible, but should be avoided.
-    In such cases, the [RP] would have no indication whether the extension was supported or processed by the client and/or
-    authenticator.
+Note: Any extensions that do not define additions to {{ClientData}} nor `authenticatorData` must be defined in a way that allows
+    the [RP] to unambiguously infer, from the returned signature or attestation statement, whether the intent of the extension
+    was followed by the client and/or authenticator. See the [[#extension-authenticator-selection]] for an example.
 
 
 ## Extending request parameters ## {#extension-request-parameters}
 
-An extension defines two request arguments. The <dfn>client argument</dfn> is passed from the <a>[RP]</a> to the client in the
-{{getAssertion()}} or {{makeCredential()}} call, while the <dfn>authenticator argument</dfn> is passed from the client to the
-authenticator during the processing of these calls.
-
-Extension definitions MUST specify the valid values for their client argument. Clients are free to ignore extensions with an
-invalid client argument. Specifying an authenticator argument is optional, since some extensions may only affect client
-processing.
+An extension defines up to two request arguments. The <dfn>client argument</dfn> is passed from the <a>[RP]</a> to the client
+in the {{getAssertion()}} or {{makeCredential()}} call, while the <dfn>authenticator argument</dfn> is passed from the client
+to the authenticator during the processing of these calls.
```

Comment:
this parag has always been ambiguous as to whether it is possible for an RP to pass an `authenticator argument` that the client then simply passes on to the authnr.  If that was an original intention of the definition of extensions, AND we intend to continue that, then lines 1453-1455 ought not be deleted.